src/app/three-examples/objects/abstract-mesh-3d.ts

```diff
@@ -0,0 +1,32 @@
+import { Input } from '@angular/core';
+import * as THREE from 'three';
+import { AbstractObject3D } from './abstract-object-3d';
+
+export abstract class AbstractMesh extends AbstractObject3D<THREE.Mesh> {
+  @Input()
+  material: string;
+  @Input()
+  materialcolor: number;
+
+  constructor() { 
+    super();
+    console.log('AbstractMesh.constructor');
+  }
+
+
+  public getMaterial(): THREE.MeshMaterialType {
+    let rmaterial: THREE.MeshMaterialType;
+    let appliedColor: number = 0xffff00; 
+    if (this.materialcolor !== undefined ) { 
+      appliedColor = this.materialcolor*1;
+    }
+    console.log('AbstractMesh.getMaterial.appliedColor: ', appliedColor);
+
+    if (this.material === "lamb" ) {
```

Comment:
TSLint should have told you to use single quotes :smile:
```suggestion
    if (this.material === 'lamb' ) {
```

Reply:
there are a lot of "s in the html...

Reply:
Yes, I think single quotes are not allowed in HTML, at least I've never seen them in the wild.